ClinkedIn/Models/InterestModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ClinkedIn.Models
+{
+    public class InterestModel
+    {
+        public InterestType Type { get; set; }
+    }
+
+    public enum InterestType
```

Comment:
Can we put the InterestType inside of the InterestModel class?

Reply:
Certainly seems we can! Done, I'll push up that change in a minute.